client/test/e2e/arc/views/app-controller-view.js

```diff
@@ -0,0 +1,38 @@
+var AppControllerView = (function () {
+  var EC = protractor.ExpectedConditions;
+
+  function AppControllerView() {
+    this.statusStoppedDisplay = element(
+      by.css('div.stopped'));
+    this.statusRunningDisplay = element(
+      by.css('div.running'));
+    this.appStartButton = element(
+      by.css('button[data-action="start"]'));
+    this.appStopButton = element(
+      by.css('button[data-action="stop"]'));
+
+    this.startStopApp = function() {
+      browser.driver.wait(
+        EC.presenceOf(this.statusStoppedDisplay),
+        10000);
+      browser.driver.wait(
+        EC.presenceOf(this.appStartButton),
+        10000);
+      this.appStartButton.click();
+      browser.driver.wait(
+        EC.presenceOf(this.statusRunningDisplay),
+        10000);
+      browser.driver.wait(
+        EC.presenceOf(this.appStopButton),
+        10000);
+      this.appStopButton.click();
+      browser.driver.wait(
+        EC.presenceOf(this.statusStoppedDisplay),
+        10000);
+    };
```

Comment:
I've been adding a `var wait = 1000;` and use that so its easy to change them all in a particular spec.